### PR TITLE
[Overflow-391] Redirect Users from accessing /manage

### PR DESCRIPTION
**Pull request by IanUrriza10**
## Backlog Link
https://framgiaph.backlog.com/view/SUN_OVERFLOW-391
## Expected Output
* Dynamic Rendering of Left Sidebar based on Admin Permissions
* Checks for permission when checking pages that start in `/manage`
## Screenshots
image
image
image
image

### CODE REVIEW

**Review by jules-lucero — DISMISSED** [review]

**Review by nicoleamber — DISMISSED** [review]

**Comment by ipilikent** [comment]
## Approved PR
- The user cannot access the manage
     - The manage page is not displayed
- The team leader can only access the teams and role page

## Proof
<details>



</details>

## Note
- I have put the issue in the bug report sheet

Test Case: https://docs.google.com/spreadsheets/d/114GrEn-LQQ8saIvjz9zqenuk05qmO4dN/edit#gid=48930172
-----
- Total Test time:  hr.
- Total Comments: 0
- Total Iterations: 0

**Comment by ipilikent** [comment]
## Approved PR
- The user cannot access the admin and team leader level pages
     - The manage page is not displayed
- The team leader can only access the teams page
     - Can't access the tags, roles, and users page
- Admin can access all manage pages
- Can go back homepage
- Dynamic Left Sidebar
- The 403 and 404 error displayed
     - The text message is correct 

## Proof
<details>

391 - FE 403 and 404

</details>

## Note
- I have put the issue in the bug report sheet

Bug Report: https://docs.google.com/spreadsheets/d/11oDnIRu6346pjNe1tz6D9xgGfio7qfBljkxug2X2YWU/edit#gid=140972773
Test Case: https://docs.google.com/spreadsheets/d/114GrEn-LQQ8saIvjz9zqenuk05qmO4dN/edit#gid=48930172
-----
- Total Test time:  1 hr.
- Total Comments: 0
- Total Iterations: 0

**Review by ipilikent — DISMISSED** [review]

**Review by ipilikent — DISMISSED** [review]

**Review by nicoleamber — DISMISSED** [review]

**Review by nicoleamber — DISMISSED** [review]

**Review by kenorenzsun — DISMISSED** [review]

**Comment by ipilikent** [comment]
## Approved PR
- The user cannot access the admin and team leader level pages
     - The manage page is not displayed
- The team leader can only access the teams page
     - Can't access the tags, roles, and users page
- Admin can access all manage pages
- Can go back homepage
- Dynamic Left Sidebar
- The 403 and 404 error displayed
     - The text message is correct 
- The background color is grey

## Proof
<details>

391 - FE 403 and 404

</details>

Test Case: https://docs.google.com/spreadsheets/d/114GrEn-LQQ8saIvjz9zqenuk05qmO4dN/edit#gid=48930172
-----
- Total Test time:  1 hr.
- Total Comments: 0
- Total Iterations: 0

**Review by ipilikent — DISMISSED** [review]

File: backend/database/seeders/PermissionSeeder.php [hunk 1]
```diff
@@ -198,16 +198,30 @@ public function run()
...
+                'name' => 'View',
+                'category' => 'User',
+                'slug' => 'view-user',
+                'description' => 'Can view users',
+            ],
+            [
+                'id' => 28,
+                'name' => 'View All',
+                'category' => 'Teams',
+                'slug' => 'viewAll-teams',
```
Comment (jules-lucero) [inline backend/database/seeders/PermissionSeeder.php:212]:
```suggestion
                'slug' => 'view-all-teams',
```

Comment (IanUrriza10) [reply backend/database/seeders/PermissionSeeder.php:212]: it would break a function i think since i factored that slugs only have 1 `-`

**Review by jules-lucero — DISMISSED** [review]

**Review by kenorenzsun — DISMISSED** [review]

**Review by kenorenzsun — APPROVED** [review]

**Review by nicoleamber — APPROVED** [review]

**Review by jules-lucero — APPROVED** [review]

**Pull request merged**